internal/observerware/common_test.go

```diff
@@ -45,13 +49,20 @@ type fakeOutbound struct {
 	err error
 }
 
-func (o fakeOutbound) Call(_ context.Context, req *transport.Request) (*transport.Response, error) {
+func (o fakeOutbound) Call(_ context.Context, _ *transport.Request) (*transport.Response, error) {
 	if o.err != nil {
 		return nil, o.err
 	}
 	return &transport.Response{}, nil
 }
 
+func (o fakeOutbound) CallOneway(_ context.Context, _ *transport.Request) (transport.Ack, error) {
+	if o.err != nil {
+		return nil, o.err
+	}
+	return fakeAck("ok"), nil
```

Comment:
It may be sufficient to use a singleton that acks with null string. That might signal better that the string is not for a message, but for a receipt over transports that need one.